internal/store/postgres/receiver.go

```diff
@@ -41,6 +41,18 @@ var receiverListQueryBuilder = sq.Select(
 	"updated_at",
 ).From("receivers")
 
+var receiverListLeftJoinSelfQueryBuilder = sq.Select(
+	"receiver.id",
+	"receiver.name",
+	"receiver.type",
+	"receiver.labels",
+	"receiver.configurations",
+	"receiver.parent_id",
+	"receiver.created_at",
+	"receiver.updated_at",
+	"parent_receiver.configurations AS parent_configurations",
+).From("receivers receiver")
```

Comment:
This is confusing, just keep it simple

```suggestion
).From("receivers r")
```